department-of-veterans-affairs/va.gov-cms
# VACMS-8682 storybook button + inputs examples

## Description
Closes #8682.
Closes #8680.

Had to move some styles from the theme into the design system in order for inputs to render as expected in Storybook. No styles have been added or removed as part of this PR, just shifted around.

Also needed to pull in a bunch of claro libraries for the same reason.... the claro libraries are **not** included in the bundled design system consumed by drupal, no worries about duplication there. Drupal's library system continues to supply the Claro libraries as needed.

## Testing done
Verified buttons and inputs rendered with expected styles in Storybook and locally in Drupal.

## Screenshots
<img width="1914" alt="Screen Shot 2022-05-04 at 1 31 52 PM" src="https://user-images.githubusercontent.com/11279744/166820951-87a978e2-4fad-4a80-9ef6-bdab83b4c021.png">
<img width="1527" alt="Screen Shot 2022-05-04 at 1 32 02 PM" src="https://user-images.githubusercontent.com/11279744/166820962-1602e782-e307-42b9-84d8-7052f6d19d71.png">

## QA steps
Visit the storybook link generated from the bot in comment below. Click the button tab, click hte input tab, see that examples are present in storybook.

Visit the CMS link generated from the bot in the comment below. See that buttons and inputs still render as expected in Drupal, no functionality should be changed.

### Definition of Done


### Select Team for PR review


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
